mcsema/BC/Lift.cpp

```diff
@@ -219,7 +219,11 @@ static InstTransResult LiftInstIntoBlockImpl(TranslationContext &ctx,
     std::cerr << "Error translating instruction at " << std::hex
               << ctx.natI->get_loc() << "; unsupported opcode " << std::dec
               << opcode << std::endl;
-    return TranslateErrorUnsupported;
+
```

Comment:
Can you guard this with a command-line option? I think using the `ignore-unsupported` would be a good option here.